src/functional/iterator/embedded.py

```diff
@@ -230,7 +230,7 @@ def __array_function__(self, func, types, args, kwargs) -> Column:
         )
 
 
-def _make_column(column_range: range, dtype=np.dtype):
+def _make_column(column_range: range, dtype: np.dtype):
     return Column(column_range.start, np.zeros(len(column_range), dtype=dtype))
```

Comment:
Unrelated, but I would just remove the function and directly construct the column.